isotovideo

```diff
@@ -53,11 +53,17 @@ use autodie ':all';
 no autodie 'kill';
 
 my $installprefix;    # $bmwqemu::scriptdir
+my $fatal_error;      # the last error message catched by the die handler
 
 BEGIN {
     # the following line is modified during make install
     $installprefix = undef;
 
+    # record the last die message
+    # note: It might *not* be a fatal error so we don't call bmwqemu::serialize_state here
```

Comment:
It seems no modules are interfering with it so far (it worked in my tests and we're already using it within the backend process along with normal signal handlers). We're not really relying on it for the correct behavior in an error case. This PR is just for additional recording of the error in case no fatal error has happened so far in any other process/component.